nipoppy/workflows/dataset_status.py

```diff
@@ -177,6 +177,19 @@ def _check_bagel(self, status_df: pd.DataFrame) -> pd.DataFrame:
         self.logger.debug(f"\tAvailable visits (n={len(session_ids)}): {session_ids}")
         self.logger.debug(f"\tAvailable pipelines (n={len(pipelines)}): {pipelines}")
 
+        # Check if at least successful run exists
+        if bagel[bagel[bagel.col_status] == STATUS_SUCCESS].empty:
+            self.logger.warning(
+                "The imaging bagel file exists but no successful run found in"
+                f" the imaging bagel file for pipeline(s): {pipelines}."
+                " If you have run a pipeline followed by 'nipoppy track', it is"
+                " likely that your pipeline output does not meet the criteria in the"
+                " '<pipeline>/tracker_config.json' file."
```

Comment:
```suggestion
                f" '{DEFAULT_LAYOUT_INFO.dpath_pipelines}/<PIPELINE_NAME>-<PIPELINE_VERSION>/tracker_config.json' file."
```
Requires that `from nipoppy.layout import DEFAULT_LAYOUT_INFO` be added to the top of the file